src/Storefront/Framework/Cache/CacheTracer.php

```diff
@@ -1,45 +0,0 @@
-<?php declare(strict_types=1);
-
-namespace Shopware\Storefront\Framework\Cache;
-
-use Shopware\Core\Framework\Adapter\Cache\AbstractCacheTracer;
-use Shopware\Core\Framework\Log\Package;
-use Shopware\Storefront\Theme\ThemeConfigValueAccessor;
-
-/**
- * @internal
- *
- * @extends AbstractCacheTracer<mixed|null>
- */
-#[Package('framework')]
-class CacheTracer extends AbstractCacheTracer
```

Comment:
parent class was deprecated, this was not. but as it is internal, I think it is okay to remove it